src/client.rs

```diff
@@ -35,6 +42,14 @@ impl DeezerClient {
         self.get_entity(id).await
     }
 
+    /// Returns the [`Album`] for Artist with the given id.
+    ///
+    /// [Deezer Api Documentation](https://developers.deezer.com/api/artist/albums)
+    pub async fn artist_albums(&self, id: u64, limit: Option<u32>,
```

Comment:
For now this should be fine, but I actually want to implement an abstraction for paging similar to the api in https://github.com/maxjoehnk/soundcloud-rs.